src/astgen.zig

```diff
@@ -2317,6 +2317,15 @@ fn compileError(mod: *Module, scope: *Scope, call: *ast.Node.BuiltinCall) InnerE
     return addZIRUnOp(mod, scope, src, .compileerror, target);
 }
 
+fn setEvalBranchQuota(mod: *Module, scope: *Scope, call: *ast.Node.BuiltinCall) InnerError!*zir.Inst {
+    try ensureBuiltinParamCount(mod, scope, call, 1);
+    const tree = scope.tree();
+    const src = tree.token_locs[call.builtin_token].start;
+    const params = call.params();
+    const target = try expr(mod, scope, .none, params[0]);
```

Comment:
Missing a type coercion to u32

Reply:
This is coerced when you do 
```zig
const quota = @truncate(u32, try resolveInt(mod, scope, inst.positionals.operand, Type.initTag(.u32)));
```

Reply:
It's better when the result location does the coercion because then something like this will work:

```zig
@setEvalBranchQuota(if (cond) 1 else 2);
```

Without the result location doing the coercion, this will give you an error.

See 

https://github.com/ziglang/zig/blob/f6644255f595fb0bbfb2fa4276804ef7031330b1/src/astgen.zig#L24-L25

and its uses